.github/integration/tests/sda/60_api_admin_test.sh

```diff
@@ -85,6 +121,13 @@ if [ -n "$output" ] ; then
     exit 1
 fi
 
+# Try to download the file that has been deleted
+resp="$(curl -s -k -L -w "%{http_code}\n" -H "Authorization: Bearer $token" -H "C4GH-Public-Key: $clientPubKey" "http://api:8080/users/test@dummy.org/file/$fileid" -o $outFile)"
```

Comment:
```suggestion
resp="$(curl -s -k -L -w "%{http_code}\n" -H "Authorization: Bearer $token" -H "C4GH-Public-Key: $clientPubKey" "http://api:8080/users/test@dummy.org/file/$fileid" -JO)"
```

Reply:
The file name "NC12878.bam.c4gh" will be used later on for uploading and test ingestion. It should not be overwritten. So better to use another file name.

Reply:
Use another file at the later stage then or use another file ID here.
This test should use the `-J -O`flags to ensure that the file is named correctly when the user doesn't supply an output filename.